aiutils/tftools/placeholder_management.py

```diff
@@ -35,8 +35,10 @@ class PlaceholderManager():
     def __init__(self):
         self._placeholders = dict()
         self.issparse = dict()
+        self.islist = dict()
```

Comment:
Why not make `islist` and `issparse` sets since you only store `islist[name] = {True, False}` and you can check `name in islist` equivalently.